Formula/pict.rb

```diff
@@ -0,0 +1,24 @@
+class Pict < Formula
+  desc "Pairwise Independent Combinatorial Tool"
+  homepage "https://github.com/Microsoft/pict/"
+  url "https://github.com/Microsoft/pict/archive/v3.7.1.tar.gz"
+  sha256 "4fc7939c708f9c8d6346430b3b90f122f2cc5e341f172f94eb711b1c48f2518a"
+
+  def install
+    system "make"
+    bin.mkpath
+    bin.install "pict"
```

Comment:
`make install` to install other files than the binary? doc, etc

Reply:
It is a very simple cli app all the info are given launching it without parameters.